### ISSUE TITLE
Rewind only works for 2 3 seconds

### ISSUE THREAD

**Issue by Darknior** [libretro/pcsx_rearmed#482]
## Description

Wehn i play other libretro emulators, like SNES, Genesis, FBNeo, i can REWIND like i want to 3 or 20 seconds without any problem.
But on PCSXRearmed (like with Duckstation, Madnafen, and on n64: Muppen64next and Parallel64) i can only REWIND 2 or 3 seconds and it stop, i can't go before. It not crash the emulator ... but 2 or 3 second is not enough ... the feature is broken :(

## Steps to reproduce

Launch a game, play 2 minutes and try to REWIND

## Your device/OS/platform/architecture

Same on RPI3b+, Odroid N2, RPI4 and PC.
All tested with Batocera.

**Comment by JustMeDaFaq** [comment]
As far as im concerned, that's expected behavior.
Basically it's not an issue with the core, but how rewinding is achieved in the frontent. The frontend needs to save the states of previous frames, which can be pretty big in more advances gaming systems like Playstation, n64, dream cast etc. So you really don't want or can't (due to ram limitations) save too many of them. That's different for systems like Snes where the size needed for that data is significantly lower and so you're able to store a lot more frames :)

**Comment by Darknior** [comment]
@JustMeDaFaq yes they explain me same on N64 core. And i understand fine.
It will take RAM ... but on a PC or an Odroin2+ it's not really a problem. Yes i PI3 will never support this, but a stronger hardware can, Not like on SNES to rewind 10 minutes lol
But maybe increasing the feature to let us rewind 10 20 seconds, it's enough in game to save us :)

**Comment by AnotherLife** [comment]
Can't the savestate interval be increased so the player can rewind more with the same number of states?

**Comment by JustMeDaFaq** [comment]
@AnotherLife Yes, that would be perfectly doable. You wouldn't even need to change the cores itself but only the frontend. So if one would be interested, one could just go to the retroarch frontends github source code and create a version with that behavior

**Comment by hizzlekizzle** [comment]
There's already the rewind granularity option, but it unfortunately still rewinds at the same number of states per second, so if you have the granularity at, say, 3, it rewinds at 3x speed.

**Comment by Darknior** [comment]
> There's already the rewind granularity option, but it unfortunately still rewinds at the same number of states per second, so if you have the granularity at, say, 3, it rewinds at 3x speed.

yes i've already try it too. It doesn't work. But the idea is fine.
